include/cocaine/rpc/actor.hpp

```diff
@@ -26,8 +26,18 @@
 
 #include <asio/ip/tcp.hpp>
 
+#include "cocaine/rpc/graph.hpp"
+
 namespace cocaine {
 
+class quote_t
+{
```

Comment:
I used this style in other places where the first line of the class definition body is one of the access specifiers.